firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/Power/AlarmsService.h

```diff
@@ -13,13 +13,23 @@
 namespace Pufferfish::Driver::Power {
 
 // All alarm codes for power management need to be registered in the following array:
-static constexpr auto alarm_codes =
-    Util::Containers::make_array<LogEventCode>(LogEventCode::LogEventCode_charger_disconnected);
+static constexpr auto alarm_codes = Util::Containers::make_array<LogEventCode>(
+    LogEventCode::LogEventCode_charger_disconnected,
+    LogEventCode::LogEventCode_battery_low,
+    LogEventCode::LogEventCode_critical_battery);
 
 class AlarmsService {
  public:
-  static void transform(
-      const MCUPowerStatus &mcu_power_status, Application::AlarmsManager &alarms_manager);
+  void transform(
+      const MCUPowerStatus &mcu_power_status, Application::AlarmsManager &alarms_manager) const;
+  static void generate_alarm(
```

Comment:
If we can make this method `private`, then we should.

Reply:
removed the method now.